server/views/pages/search.njk

```diff
@@ -33,49 +33,55 @@
               <div class="moj-action-bar__filterTagsContainer govuk-!-margin-bottom-5">
               </div>
 
-              {% if searchResults.length %}
+              {% if searchResults %}
                 <h2 class="govuk-heading-l" data-qa="search-results-count">{{ paginationParams.results.count }} Prisoners</h2>
-
                 {% if searchResults.length %}
+
                   {{ mojPagination(paginationParams) }}
-                {% endif %}
 
-                {% set rows = [] %}
-                {% for prisoner in searchResults %}
-                  {% set cellHtml %}
-                    <div class="govuk-grid-column-three-quarters">
-                      <div class="govuk-body-lead govuk-!-font-weight-bold govuk-!-margin-bottom-1">
-                        <a href = '/detail/{{ prisoner.prisonNumber }}'>{{ prisoner.firstName | capitalize }} {{ prisoner.lastName }}</a> &middot; {{ prisoner.dob | formatDate }}
-                      </div>
-                      <div class="govuk-body-m govuk-!-margin-bottom-1">Prison no. {{ prisoner.prisonNumber }} &middot;
-                        Reception {{ prisoner.receptionDate | formatDate }}
-                      </div>
-                      {% if prisoner.isAlias %}
-                        <div class="govuk-body govuk-!-margin-bottom-1" data-qa="alias">
-                          Matched on alias {{ prisoner.aliasFirst }} {{ prisoner.aliasLast }}
+                  {% set rows = [] %}
+                  {% for prisoner in searchResults %}
+                    {% set cellHtml %}
+                      <div class="govuk-grid-column-three-quarters">
+                        <div class="govuk-body-lead govuk-!-font-weight-bold govuk-!-margin-bottom-1">
```

Comment:
Feels like a macro might be good here

Reply:
good idea - just been looking at those actually